### PR TITLE
Nr 89185 Vertica

### PR DESCRIPTION
**Pull request by pkosanam**
Ticket: https://issues.newrelic.com/browse/NR-89185
Dashboard: https://onenr.io/07j9YYqDVjO

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/vertica/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+
+slug: vertica
+description: |
+  ## Comprehensive monitoring quickstart for Vertica
```
Comment (mdumpati) [inline quickstarts/vertica/config.yml:5]: @pkosanam Can you please add "Why monitor Veritca?" here.

File: dashboards/vertica/vertica.json [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+            "row": 2,
+            "width": 2,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details).\n\nFollow New Relic [Vertica documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/vertica-monitoring-integration/) to instrument Vertica\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=Vertica) and let us know how we can improve it for you.\n"
```
Comment (mdumpati) [inline dashboards/vertica/vertica.json:162]:
```suggestion
            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details).\n\nFollow [Vertica documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/vertica-monitoring-integration/) to instrument Vertica\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=Vertica) and let us know how we can improve it for you.\n"
```

File: dashboards/vertica/vertica01.png
Comment (mdumpati) [inline dashboards/vertica/vertica01.png:1]: @pkosanam After changing the About text replace the Image as well.

**Review by mdumpati — COMMENTED** [review]
@pkosanam Added comments

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Pull request merged**